kernel/README.md

```diff
@@ -68,3 +68,13 @@ We use [GitHub Issues](https://github.com/delta-io/delta/issues) to track commun
 
 # Contributing
 We welcome contributions to Delta Lake and we accept contributions via Pull Requests. See our [CONTRIBUTING.md](https://github.com/delta-io/delta/blob/master/CONTRIBUTING.md) for more details. We also adhere to the [Delta Lake Code of Conduct](https://github.com/delta-io/delta/blob/master/CODE_OF_CONDUCT.md).
+
+# Setting up IDE
+
+Java code adheres to the [Google style](https://google.github.io/styleguide/javaguide.html), which is verified via `build/sbt javafmtCheckAll` during builds.
```

Comment:
let's say and use `build/sbt kernelGroup/javafmtCheckAll` and `build/sbt kernelGroup/javafmtAll` instead (i.e. specify kernelGroup, don't just run this against the root)

Reply:
IMO this is just more text to type for a Dev. Given that all other projects are being ignored there's no harm running this against the root project. Pretty much all projects that I've been involved in run this against the root and exclude the formatting from being applied on subprojects.
I can still update the wording if you think it's worth it, but people will typically just type the shortest command to get the code formatted